lld/test/ELF/riscv-relax-call-mixed-rvc.s

```diff
@@ -0,0 +1,30 @@
+# REQUIRES: riscv
+# RUN: rm -rf %t && split-file %s %t && cd %t
+# RUN: llvm-mc -filetype=obj -triple=riscv64 -mattr=+c,+relax a.s -o a.o
+# RUN: llvm-mc -filetype=obj -triple=riscv64 -mattr=-c,+relax b.s -o b.o
+
+# RUN: ld.lld a.o b.o --shared -o a
+# RUN: llvm-objdump -d --no-show-raw-insn -M no-aliases a | FileCheck %s
+
+## This needs to be a *uncompressed* jal instruction since it came from the
+## source file which does not enable C
+# CHECK-LABEL: <foo>:
+# CHECK-NEXT:    1260: jal zero, 0x1260 <foo>
```

Comment:
Omit `1260:` so that the test is less sensitive to address changes.

`jal zero, {{.*}} <foo>`

Reply:
I need the addresses to make the instruction size difference obvious.  Is there a way to force the address so it's stable instead?

Reply:
Consider `-Ttext=0x10000` as many `riscv-relax-*` files use.